### ISSUE TITLE
3.0 custom i18n-Tables stopped working

### ISSUE THREAD

**Issue by Soswald** [cakephp/cakephp#5659]
When trying to get a list including translations via
```php
$this->set('countries', $this->paginate($this->Countries->find('translations')));
```
it fails with
```
SQLSTATE[42S02]: Base table or view not found: 1146 Table '**DATABASE**.countries-i18n' doesn't exist
```
for my CountriesTable, which is initialized with
```php
    public function initialize(array $config) {
        $this->addBehavior('Translate', [
            'fields' => ['name'],
            'translationTable' => 'countries_i18n'
        ]);
        $this->table('countries');
        $this->displayField('name');
        $this->primaryKey('id');
    }
```

which points to a MySQL table, set up according to the book.

The error page states
```
Please try correcting the issue for the following table aliases:

countries_i18n
Countries_name_translation
countries-i18n
```
I think `countries-i18n` should not be there.

The last statement in the SQL-Log is:
```sql
SHOW FULL COLUMNS FROM `countries-i18n`
```
But this table doesn't exist, nor should it exist I think, because I set the translations table to be `countries_i18n`, which seems to work, acording to the first statement in the SQL-log is taken correctly.
```sql
SELECT Countries.id AS `Countries__id`, Countries.name AS `Countries__name`, 
Countries.code AS `Countries__code`, Countries.fright_rate AS `Countries__fright_rate`, 
Countries.labour_rate AS `Countries__labour_rate`, 
Countries_name_translation.id AS `Countries_name_translation__id`, 
Countries_name_translation.locale AS `Countries_name_translation__locale`, 
Countries_name_translation.model AS `Countries_name_translation__model`, 
Countries_name_translation.foreign_key AS `Countries_name_translation__foreign_key`, 
Countries_name_translation.field AS `Countries_name_translation__field`, 
Countries_name_translation.content AS `Countries_name_translation__content` 
FROM countries Countries 
LEFT JOIN countries_i18n Countries_name_translation 
ON (Countries_name_translation.model = 'Countries' AND Countries_name_translation.field = 'name' AND Countries_name_translation.locale = 'de' AND Countries.id = (Countries_name_translation.foreign_key)) LIMIT 20 OFFSET 0
```
If more information or logs are needed, I be glad to assist.

**Comment by lorenzo** [comment]
We decided to make a change that sadly had this effect. It is easy to fix, though. Change:

Create the class for your Translations table:

```sh
bin/cake bake CountriesI18n
```

Associated the Behavior to that Table:
```php
$this->addBehavior('Translate', [
            'fields' => ['name'],
            'translationTable' => 'CountriesI18n'
        ]);
```

**Comment by Soswald** [comment]
Thanks working again.

For others searching this it is `bin/cake bake model *Model*I18n`

**Comment by Xety** [comment]
You just need to change `'translationTable' => 'countries_i18n'` by `'translationTable' => 'CountriesI18n'`.

**Comment by Soswald** [comment]
Accidental closed...

**Comment by lorenzo** [comment]
I suggest the model to be baked, though. That way you can control things like the default connection name.

**Comment by markstory** [comment]
I can update the documentation for this.

**Comment by markstory** [comment]
Fixed in cakephp/docs@bf864f339b12e57da1a27b78298e7b75d0c18867
